### ISSUE TITLE
support zigbee connected plugs via gateways

### ISSUE THREAD

**Issue by BonRm** [Bluebie/homebridge-miio#8]

**Comment by Bluebie** [comment]
Nope! It only works with the WiFi plugs. I can't really support devices I don't have access to. If you can find any info on how to interface with those, I'd be very happy to add support though!

**Comment by BonRm** [comment]
I have this plugin installed https://github.com/snOOrz/homebridge-aqara to use zigbee power plugs but all the other accessories are working except the power plugs. thanks anyway

**Comment by astrojetson8** [comment]
@BonRm even though they are banded as xiaomi products. this is purely for marketing, and they are made by different manufactures that are part of xiaomi eco system. 

@Bluebie https://github.com/louisZL/lumi-gateway-local-api
as i understand homebridge-aqara is using Lumi API

**Comment by Bluebie** [comment]
A recent update to the underlying miio library might make it possible to support this in the future, so i'm reopening the issue.

**Comment by aholstenson** [comment]
I'm thinking about updating the discovery mechanism a bit to automatically discover devices on a Smart Home Gateway which should make this a lot easier to support. I'll see when I can fit those changes in.
